WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -265,6 +274,7 @@ extension ProductFormViewController: TextViewViewControllerDelegate {
 private extension ProductFormViewController {
     func editProductDescription() {
         let editorViewController = EditorFactory().productDescriptionEditor(product: product) { [weak self] content in
+            ServiceLocator.analytics.track(.aztecEditorDoneButtonTapped)
```

Comment:
⚠️ I think this should be in `AztecEditorViewController` instead. Reasons:

- The event name is generic. It mentions “aztec” and not ”product description”. 
- Since it is generic, it's probably more future-proof if we place it inside `AztecEditorViewController` in case `AztecEditorViewController` is used somewhere else. This is how we did it [in WCAndroid](https://github.com/woocommerce/woocommerce-android/blob/9ad40e8d1b086d3c61abe8cc60f1d9315a028bde/WooCommerce/src/main/kotlin/com/woocommerce/android/ui/aztec/AztecEditorFragment.kt#L85).

Reply:
Thanks @shiki for noticing it!
I don't know if we want to know this information is relevant. Do we want to know if the done button on Aztec editor was tapped?
As mentioned above, it would be appropriate to rename the event, otherwise, we would lose tracking when the product description is edited.